src/styles/blocks/button.scss

```diff
@@ -0,0 +1,11 @@
+@use '../_variables/variables' as *;
+
+.button {
+  border: none;
+  border-radius: 5px;
+  padding: 12px 0;
+  text-align: center;
+  font-size: 14px;
+  font-weight: $FontBold;
```

Comment:
The variable `$FontBold` should be imported from the variables file. Ensure that the variable name matches the one defined in `variables.scss`.